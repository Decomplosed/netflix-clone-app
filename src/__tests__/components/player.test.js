import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import { Player } from '../../componentsnts';

describe('<Player />', () => {
  it('Renders the <Player /> with a bunny video', () => {
    const { container, getByText, queryByTestId } = render(
      <Player>
        <Player.Button>
          <Player.Video src='/videos/bunny.mp4' />
        </Player.Button>
      </Player>,
    );
  });
});
