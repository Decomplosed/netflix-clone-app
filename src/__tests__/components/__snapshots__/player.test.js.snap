// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Player /> renders the <Player /> with a bunny video 1`] = `
<div
  class="player__Container-sc-14h7g0h-0"
>
  <button
    class="player__Button-sc-14h7g0h-4 kLcuGw"
  >
    Play
  </button>
</div>
`;
